openstack/identity/v3/projects/requests_test.go

```diff
@@ -0,0 +1,62 @@
+package projects
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/rackspace/gophercloud/testhelper"
+	"github.com/rackspace/gophercloud/testhelper/client"
+)
+
+func TestCreateSuccessful(t *testing.T) {
+	testhelper.SetupHTTP()
+	defer testhelper.TeardownHTTP()
+
+	testhelper.Mux.HandleFunc("/projects", func(w http.ResponseWriter, r *http.Request) {
+		testhelper.TestMethod(t, r, "POST")
+		testhelper.TestHeader(t, r, "X-Auth-Token", client.TokenID)
+		testhelper.TestJSONRequest(t, r, `{
+		  "project": {
+		    "description": "My new project",
+		    "domain_id": "default",
+		    "enabled": true,
+		    "is_domain": true,
+		    "name": "myNewProject"
+		  }
+		}`)
+
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusCreated)
+		fmt.Fprintf(w, `{
+            "project": {
```

Comment:
is this the standard indentation under the fmt line or should it be indented further?  kindof a nit sorry ;).